### ISSUE TITLE
[bug] [themes]: Dracula Theme Color Scheme not respected

### ISSUE THREAD

**Issue by Bellisario** [ghostty-org/ghostty#3441]
### TL;DR

The current built-in Dracula theme doesn't respect the official Dracula scheme, uses different colors and is missing light-versions of colors (reusing "non-light" versions), presenting contrast issues.

## What I've experienced

I have just installed Ghostty and I chose to use the Dracula color scheme, as I did with Kitty.\
Just after the first seconds I tried using `zsh-autosuggestions` but saw the autocompletion color in Ghostty was different than the one in Kitty (and the one in Ghostty had really low contrast with the background).

Kitty (left) vs Ghostty (right):
<img width="842" alt="Image" src="https://github.com/user-attachments/assets/b1e7b01d-9fae-42d6-b2fc-99728f100bb4" />

After investigating a bit, I discovered that the colors used by Kitty's Dracula theme are (mostly) different in Ghostty. For example, the autocompletion color used is the "8th color", which is `#6272a4` for Kitty, while `#555555` for Ghostty.

Anyway, I'm certain the "wrong" color scheme is Ghostty's because Kitty uses the color scheme from [the official Dracula (org) repo](https://github.com/dracula/kitty/blob/master/dracula.conf), while Ghostty seems to use [iTerm 2 "unofficial" theme](https://github.com/mbadolato/iTerm2-Color-Schemes/blob/master/ghostty/Dracula).

Here are the (comparable) schemes used by Kitty and Ghostty, respectively (from the links above):

> Light versions of colors are completely reused from non-light ones, some colors are different from the ones used by Kitty (ex. background)

<table>
<tr>
<td> Kitty </td> <td> Ghostty </td>
</tr>
<tr>
<td>

```
foreground            #f8f8f2
background            #282a36
selection_foreground  #ffffff
selection_background  #44475a

# black
color0  #21222c
color8  #6272a4

# red
color1  #ff5555
color9  #ff6e6e

# green
color2  #50fa7b
color10 #69ff94

# yellow
color3  #f1fa8c
color11 #ffffa5

# blue
color4  #bd93f9
color12 #d6acff

# magenta
color5  #ff79c6
color13 #ff92df

# cyan
color6  #8be9fd
color14 #a4ffff

# white
color7  #f8f8f2
color15 #ffffff

# Cursor colors
cursor            #f8f8f2
cursor_text_color background
```

</td>
<td>

```
foreground = #f8f8f2
background = #1e1f29
selection-foreground = #ffffff
selection-background = #44475a

# black
palette = 0=#000000
palette = 8=#555555

# red
palette = 1=#ff5555
palette = 9=#ff5555

# green
palette = 2=#50fa7b
palette = 10=#50fa7b

# yellow
palette = 3=#f1fa8c
palette = 11=#f1fa8c

# blue
palette = 4=#bd93f9
palette = 12=#bd93f9

# magenta
palette = 5=#ff79c6
palette = 13=#ff79c6

# cyan
palette = 6=#8be9fd
palette = 14=#8be9fd

# white
palette = 7=#bbbbbb
palette = 15=#ffffff

cursor-color = #bbbbbb


```

</td>
</table>

Haven't looked at other themes but I hope they respect the original theme scheme... in my case the only flaw of migrating from Kitty to Ghostty was just the theme.

## Throughts

From what I've experienced, I really liked the way Ghostty let you choose a theme, but what I really feel missing is support of official themes (ex. Dracula, Cappuccin), that should be in the "Ghostty org" or in the "theme org", not a "third-party org".

**Comment by mitchellh** [comment]
Upstream: https://iterm2colorschemes.com

**Comment by Gelmo** [comment]
Just posted about this here - https://github.com/mbadolato/iTerm2-Color-Schemes/issues/487

**Comment by Gelmo** [comment]
@Bellisario @mitchellh This has now been resolved upstream, if you want to run that workflow manually

**Comment by mitchellh** [comment]
Thanks, I'll do so before the next release, otherwise the weekly job will get it.

**Comment by Gelmo** [comment]
Now that I've tested this, it unfortunately is not resolved. More info here - https://github.com/mbadolato/iTerm2-Color-Schemes/pull/490#issuecomment-2568280741
